test/e2e/crypta_suite_test.go

```diff
@@ -0,0 +1,26 @@
+package crypta_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
```

Comment:
Wouldn't it be cleaner (at least for the gexec) module to import it with a qualifier?
I guess for the root gomega and ginko it probably makes sense because it is structure related.
What do you think?

Reply:
True, thanks

Reply:
I can already remember again why I directly imported gexec:
![grafik](https://github.com/codetent/crypta/assets/2047922/2efb244d-b209-42bf-84cc-c6a4655c93a3)
Let me know what you think. I like that the wording nicely integrates with the assertions without any qualifiers, but if you have any doubts let me know and we can remove the direct import.

Reply:
I think it easier to understand where such commands like `Build` or `Start` are coming from since they sound very generic. Therefore, I would add the `gexec` name in front for the process matchers. Otherwise, it will be hard to understand (for someone who does not know the code or libraries) where the functions are defined.